modules/button/css/README.md

```diff
@@ -25,6 +25,26 @@ Add your `node_modules` directory to your SASS `includePaths`. You will then be
 [postcss-inline-svg](https://github.com/TrySound/postcss-inline-svg) plugin to properly process and
 inline icons. Process your SASS through PostCSS once it has been compiled to CSS.
 
+## Deprecation Warning
+
+> We are rolling out our next iteration of our buttons and will be deprecating the old ones(orange
+> primary, secondary, and delete) on (this time frame).
+
+> We've renamed the old buttons to use `wdc-button-deprecated` mixin and have added a warning
```

Comment:
Most people don't use the mixins for components and there's no warning associated with using the class `wdc-btn-deprecated`. I would just remove this messaging.